packages/fxa-shared/test/db/models/auth/sp_accountDevices.sql

```diff
@@ -0,0 +1,41 @@
+CREATE PROCEDURE `accountDevices_16` (
```

Comment:
I don't have a good solution for this, but updates to auth-server procedure would mean this needs to be updated as well.